src/main/java/org/spongepowered/common/world/gen/SpongeChunkGenerator.java

```diff
@@ -504,6 +505,10 @@ public void recreateStructures(Chunk chunkIn, int x, int z) {
             ((IChunkGenerator) this.baseGenerator).recreateStructures(chunkIn, x, z);
             return;
```

Comment:
I think this return might be an error, preventing plugin structures from working properly.

Reply:
That return was there prior to my changes.  I am not sure I the setup to see what would happen if I removed it.

How shall I proceed?

Reply:
Yes I know, but the structures covered by the (effectively) three cases will all be exclusive.